src/render.jl

```diff
@@ -5,28 +5,38 @@ parsedocs(ds::Docs{:md}) = Markdown.parse(data(ds))
 ## Common -------------------------------------------------------------------------------
 const MDHTAGS = ["#", "##", "###", "####", "#####", "######"]
 const MDSTYLETAGS = ["", "*", "**"]
+const MD_SUBHEADER_OPTIONS = [:SKIP, :SIMPLE, :CATEGORY]
 
 type Config
-    # General Options
-    include_internal :: Bool
-    # Html only Options
-    mathjax          :: Bool
-    # MarkDown only Options
-    mdstyle_header   :: ASCIIString
-    mdstyle_objname  :: ASCIIString
-    mdstyle_meta     :: ASCIIString
-    mdstyle_exported :: ASCIIString
-    mdstyle_internal :: ASCIIString
+    category_order         :: Vector{Symbol}
+    include_internal       :: Bool
+    mathjax                :: Bool
+    mdstyle_header         :: ASCIIString
+    mdstyle_objname        :: ASCIIString
+    mdstyle_meta           :: ASCIIString
+    mdstyle_subheader      :: ASCIIString
+    mdstyle_genindex_mod   :: ASCIIString
+    md_subheader           :: Symbol
+    md_genindex            :: Bool
+    md_genindex_modprefix  :: ByteString
```

Comment:
`ByteString` -> `UTF8String` to make all type fields concrete.

`ByteString` is an alias for `Union(ASCIIString, UTF8String)`.